src/main/java/org/codehaus/plexus/archiver/zip/AbstractZipArchiver.java

```diff
@@ -212,6 +223,14 @@ public boolean isFilesonly()
         return doFilesonly;
     }
 
+    public Long getFixedEntryModificationTime() {
+        return fixedEntryModificationTime;
+    }
+
+    public void setFixedEntryModificationTime(Long fixedEntryModificationTime) {
```

Comment:
Summarizing:

- a method `setModificationTime(Date modificationTime)` on `Archiver` (+getter)
- a field `private Date forcedModificationTime = null` on `AbstractArchiver`
- The field is only used by TarArchiver, AbstractZipArchiver and
- Question then: do we also need overloaded methods for addFile/addResource to allow overriding the modificationTime on an entry basis (and having a `modificationTime` field in `ArchiveEntry`)?